modules/wp-rest/classes/elementor-post-meta.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Elementor\Modules\WpRest\Classes;
+
+use Elementor\Plugin;
+use Elementor\Utils;
+
+defined( 'ABSPATH' ) || exit;
+
+class ElementorPostMeta {
+
+	public function register() :void {
+		$post_types = get_post_types_by_support( 'elementor' );
+
+		foreach ( $post_types as $post_type ) {
+			register_meta($post_type, '_elementor_edit_mode', [
+				'single' => true,
+				'show_in_rest' => [
+					'schema' => [
+						'title' => 'Elementor edit mode',
+						'description' => 'Elementor edit mode, `builder` is required for Elementor editing',
+						'type' => 'string',
+						'enum' => [ '', 'builder' ],
+						'context' => [ 'edit' ],
+					],
+				],
+				'auth_callback' => [ $this, 'check_edit_permission' ],
+			]);
+
+			$document_types = Plugin::$instance->documents->get_document_types();
+
+			register_meta($post_type, '_elementor_template_type', [
+				'single' => true,
```

Comment:
It seems odd. Can we avoid adding all this new meta?

Reply:
How will it be integrated into the wp-rest scheme?